### ISSUE TITLE
Demo Day presentation Deck

### ISSUE THREAD

**Issue by andrewsgreene** [defenseunicorns/baffles-matlab-demo#8]
As an interested party of Baffles, I want to view a presentation, So that I understand the value proposition of Baffles.

## Acceptance Criteria
Given Demo Day
When presenting
Then We have a Baffles presentation deck

**Comment by codywyoming177** [comment]
https://docs.google.com/presentation/d/1p-Pf75ELi5SF1_m5KzpQgMSlXXGZEb91c4JIo6XlYPs/edit?pli=1#slide=id.g122068f30e7_0_693
